### DISCUSSION TITLE
Dashboard doesn't display details.

### DISCUSSION THREAD

**Discussion by devaniranjan** [chaoss/grimoirelab#682] [Q&A]
I am using docker-compose to create the dashboard. It works but for the organization, I can't see details. It says 'unknown'. 
![Screenshot 2024-07-26 at 10 51 09 AM](https://github.com/user-attachments/assets/dd40f2d4-2743-44d3-83e4-8db37dc17fd7)

What am I doing wrong that the repositories details are not being pulled? 

![Screenshot 2024-07-26 at 10 50 29 AM](https://github.com/user-attachments/assets/ce84776c-5feb-4886-ab37-d66827d67344)
Also seeing some data but for others it says no data found [which is not correct].

**Comment by sduenas** [comment]
To get data about organizations you should log in on the identities interface (`/identities`) and add organizations there and their domains. Matching of identities and organizations will be done taking into account the domain of the email address. 

For the pull requests and issues, you should configure to retrieve data from there too. See [the tutorial for more info](https://chaoss.github.io/grimoirelab-tutorial/docs/data-sources/configurations/#pull-request)

https://github.com/chaoss/grimoirelab/blob/master/default-grimoirelab-settings/setup.cfg#L68

**Comment by devaniranjan** [comment]
@sduenas I can't even get the default install of docker-compose to pull organizations (I didn't change anything in the docker container). 
![Screenshot 2024-07-29 at 9 04 26 AM](https://github.com/user-attachments/assets/088c135b-2aa3-4644-8fa4-46299771622b)


I get the following. 
![Screenshot 2024-07-29 at 9 04 55 AM](https://github.com/user-attachments/assets/216318af-b1f2-455f-b297-5d7a1e3860ce)


Thank you on the pull/issues. I was able to resolve that. 

I will close this to avoid duplication of effort, since i also created an issue from this discussion earlier.
